### PR TITLE
Improve maths of inferring inclination

### PR DESCRIPTION
**Pull request by rikvl**

### CODE REVIEW

File: docs/tutorials/infer_phys_pars.rst [hunk 1]
```diff
@@ -627,43 +627,52 @@ Pulsar orbital inclination
...
     \cos^2( \chi_\mathrm{p} ) \sin^4( i_\mathrm{p} )
         - ( 1 + Z^2 ) \sin^2( i_\mathrm{p} ) + Z^2 = 0,
     \qquad \mathrm{with} \qquad
-    Z = \frac{ v_{0,\oplus} K_\mathrm{p} b_\oplus }
+    Z = \frac{ \sin( i_\mathrm{p} ) }{ b_\mathrm{p} }
+      = \frac{ v_{0,\oplus} K_\mathrm{p} b_\oplus }
              { A_\oplus A_\mathrm{p} d_\mathrm{p} }.
 
-The standard quadratic formula then gives the solutions
+Here, the auxiliary parameter :math:`Z` is restricted to the range
```
Comment (mhvk) [inline docs/tutorials/infer_phys_pars.rst:642]: Is it really useful to discuss the range in `Z` with `cos(\chi)=1` special-cased?

**Review by mhvk — COMMENTED** [review]
This looks good with the possible exception of the in-line comment.

File: docs/tutorials/infer_phys_pars.rst (hunk 1 above)
Comment (rikvl) [reply docs/tutorials/infer_phys_pars.rst]: Removed. Indeed this was more something that came up analyzing the properties of that equation, and it's not very useful for the tutorial.

**Review by mhvk — APPROVED** [review]
OK, thanks! Let's get it in.

**Pull request merged**